basics/type-qualifiers.md

```diff
@@ -0,0 +1,111 @@
+# Typqualifizierer
+
+D besitzt eine statische Typisierung. Wenn eine Variable deklariert wurde, kann der
+Typ nicht mehr verändert werden. Dies erlaubt dem Kompiler viele Fehler früh
+zu erkennen und Restriktionen können schon während der Kompilierung erkannt werden.
+Ein gutes Typsystem leistet die benötigte Unterstützung für die Sicherheit
+und bessere Wartbarkeit großer Programme.
+
+D stellt verschiedene Typqualifizierer zur Verfügung aber die meist Genutzten sind
+`const` und `immutable`.
+
+### `immutable`
+
+Zusätzlich zu der statischen Typisierung, stellt D Typqualifizierer (auch "Speicherklassen" genannt)
+bereit welche weitere Beschränkungen erlauben. Zum Beispiel kann ein `immutable` (unveränderbares)
+Objekt nur einmal initialisiert werden und kann danach nicht mehr verändert werden.
+
+    immutable int err = 5;
+    // oder: immutable arr = 5 (der Typ int wird automatisch inferriert)
+    err = 5; // FEHLER
+
+Wegen ihrer Unveränderbarkeit, können `immutable` Objekte sicher und ohne Syn­chro­ni­sa­ti­on
+zwischen Threads geteilt werden. Dies impliziert auch, dass `immutable` Objekte perfekt
+gecacht werden können.
+
+### `const`
+
+`const` Objekte können ebenfalls nicht verändert werden. Diese Restriktion ist
+jedoch nur für den aktuellen Geltungsbereich gültig. Ein `const` Zeiger
+kann sowohl von einem `immutable` Objekt als auch von einem
```

Comment:
Um den gleichen Stil in beiden Satzteilen zu bewahren würde ich "unveränderbar" auch erwähnen:

> unveränderbaren (`immutable`)